modules/gitops-release-notes-1-7-1.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+:_content-type: REFERENCE
+
+[id="gitops-release-notes-1-7-1_{context}"]
+= Release notes for {gitops-title} 1.7.1
+
+{gitops-title} 1.7.1 is now available on {product-title} 4.8, 4.9, 4.10, 4.11, and 4.12.
+
+[id="errata-updates-1-7-1_{context}"]
+== Errata updates
+
+=== RHSA-2023:0467 - {gitops-title} 1.7.1 security update advisory 
+
+Issued: 2023-01-25
+
+{gitops-title} release 1.7.1, which includes security updates, is now available. The list of security fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2023:0467[RHSA-2023:0467] advisory. 
+
+If you have installed the {gitops-title} Operator, run the following command to view the container images in this release:
+
+[source,terminal]
```

Comment:
Will this command change with each version?

Reply:
@iam-veeramalla, Could you please provide your inputs here? Also, I added this command to make it look consistent with the format we followed in the OCP RN here: https://docs.openshift.com/container-platform/4.11/release_notes/ocp-4-11-release-notes.html#ocp-4-11-2

Reply:
it will be constant.